packages/api/src/external/carequality/command/cq-directory/create-or-update-cq-organization.ts

```diff
@@ -1,98 +1,79 @@
-import NotFoundError from "@metriport/core/util/error/not-found";
+import { errorToString, BadRequestError, metriportCompanyDetails } from "@metriport/shared";
 import { CarequalityManagementAPI } from "@metriport/carequality-sdk";
 import { isOboFacility } from "../../../../domain/medical/facility";
 import { OrganizationModel } from "../../../../models/medical/organization";
 import { FacilityModel } from "../../../../models/medical/facility";
 import { out } from "@metriport/core/util/log";
 import { capture } from "@metriport/core/util/notifications";
-import { errorToString } from "@metriport/shared";
 import { makeCarequalityManagementAPI } from "../../api";
 import { CQOrganization } from "../../organization";
-import { CQOrgDetails, getCqAddress, getParsedCqOrgOrFail } from "../../shared";
+import { CQOrgDetails, getCqAddress, getParsedCqOrgOrFail, getCqOrg } from "../../shared";
 import { metriportIntermediaryOid, metriportOid } from "./create-or-update-cq-facility";
 import { metriportEmail as metriportEmailForCq } from "../../constants";
-import { metriportCompanyDetails } from "@metriport/shared";
 
 const cq = makeCarequalityManagementAPI();
 
 export async function createOrUpdateCQOrganization(
-  orgDetails: CQOrgDetails,
-  active: boolean
```

Comment:
why are we removing the param for `active`? IIRC, we use this to deactivate cx org on CQ

Reply:
This is now only being passed as part of the org's payload.

Gotta test if we can make an inactive CQ dir entry active by sending an update w/ this flag set to true. Otherwise, we'll prob need to adjust the logic or cancel the operation if the org exists but it's inactive.